### PR TITLE
Masks blog post

### PR DESCRIPTION
**Pull request by mattbell**
Adding the masks blog post.  I've tested it locally.

### CODE REVIEW

**Review by beshaya — COMMENTED** [review]
Could you update the graph based on published numbers? (specifically P100 + googles didn't make the cut)

File: src/posts/blog/6-masks/masks.ts [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+
+## Surgical masks
+
+* **Protection of you:** Moderate
+* **Protection of others:** Good
+* **Fit quality:** Mediocre
+* **Breathability:** Good
+* **Useful for:** Outdoor activities
+* **Benefits:** Cheap
+* **Downsides:** Disposable, limited protection, can’t get wet.
```
Comment (beshaya) [inline src/posts/blog/6-masks/masks.ts:119]: I thought we agreed to remove the wet part? Maybe "can't be washed"?

File: src/posts/blog/6-masks/masks.ts [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+*   The nose bridge and the bottom of the chin are where fit is most often an issue.  Any good N95 will have a wire or metal strip on the nose bridge so that it will fit your nose.
+*   The mask should have straps that go around the back of your head, not just ear loops.  Head straps give you a tighter fit
+
+### Reusing your N95
+*   You’ll need to throw out your N95 mask if it gets wet, overly dirty, or difficult to breathe through.
```
Comment (beshaya) [inline src/posts/blog/6-masks/masks.ts:183]: Could this be worded less aggressively? A boiled N95 mask is still better than anything earlier on this list.

File: src/posts/blog/6-masks/masks.ts [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+
+**Exhalation valves:** It should be noted that some N95s come with an exhalation valve.  While this makes the mask easier to breathe through, it lets you expel air from your mouth without any filtration, which isn’t safe for the people around you.  However, you don’t need to throw these masks out; you can just tape over the exhalation valve (on the outside, so other people know) and then wear it like a normal N95, or just put a surgical mask over it.
+
+<figure>
+<img src=${exhalationvalvetape} width="531", height="258"/>
+</figure>
+
+**How N95s work:** N95 (and KN95) masks work on a very different principle from cloth masks -- they have an electrostatic charge that attracts small airborne particles that pass through the mask.  Water destroys the electric charge, so getting the mask wet will make it stop working.  Separately, the mask will stop working effectively once too many particles have become caught in it.  Here’s a [cute video explanation](https://www.youtube.com/watch?v=eAdanPfQdCA).
```
Comment (beshaya) [inline src/posts/blog/6-masks/masks.ts:193]: Still no source for wetting being a problem. Prefer "getting the mask wet may degrade its performance"

File: src/posts/blog/6-masks/masks.ts (hunk 1 above)
Comment (mattbell) [reply src/posts/blog/6-masks/masks.ts:119]: I'll go with "moisture degrades performance"

File: src/posts/blog/6-masks/masks.ts (hunk 2 above)
Comment (mattbell) [reply src/posts/blog/6-masks/masks.ts:183]: Yeah I can do that.

File: src/posts/blog/6-masks/masks.ts (hunk 1 above)
Comment (mattbell) [reply src/posts/blog/6-masks/masks.ts]: > Could you update the graph based on published numbers? (specifically P100 + googles didn't make the cut)

Done

**Review by blanchardjeremy — APPROVED** [review]
LGTM

**Pull request merged**